src/Tags.elm

```diff
@@ -118,18 +132,65 @@ dataExtractor =
     .tags >> Set.toList >> Encode.list Encode.string
 
 
-declarationVisitor : Node Declaration -> ModuleContext -> ( List never, ModuleContext )
-declarationVisitor node ({ tags } as context) =
+moduleDefinitionVisitor : Node Module -> ModuleContext -> ( List never, ModuleContext )
```

Comment:
What are all the `List Never` for?

Reply:
It's kind of required, because I only output empty lists. But looking it from the other way, it signals from the type signature that those list must be empty, since you cannot create a generic type out of nowhere (I called it `never`, but it could have been anything, `void` is another nice one).

Reply:
I take it the elm review api requires you return the `List something`? never seems like a good choice for the name, it matches the Never type.

Reply:
At first I read your sentence as "it never seems like a good choice for the name" hahaha :D

I think I tried using `Never` before, but I vaguely remember it didn't work? I'll try again and clarify.